src/telegraf.ts

```diff
@@ -135,18 +141,22 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
     })
   }
 
-  private startWebhook(
-    hookPath: string,
-    tlsOptions?: TlsOptions,
-    port?: number,
-    host?: string,
-    cb?: http.RequestListener
-  ) {
+  private startWebhook(config: {
```

Comment:
Wary of breaking change in a minor release, can we just accept `secretToken` as the last param for now and improve on it in v5?

Reply:
Do you think it is necessary to leave it as it was?
The truth is that for my way of using Telegraf it is easier for me to use this method if only an object is passed as the only argument in case the Telegraf class is extended


```javascript
class NewTelegrafClass extend Telegraf {
  constructor(...) {
    super(...)
  }
  startWebhook({ anotherOption }) {
  }
}
 ```

Reply:
Yes, I completely agree the object way is better; but it will break existing user code, which we must avoid in a non-major version. That's why I said we should do this change in v5